extensions/arc/src/ui/dialogs/connectControllerDialog.ts

```diff
@@ -116,6 +126,8 @@ export class ConnectToControllerDialog extends InitializingComponent {
 		try {
 			// Validate that we can connect to the controller
 			await controllerModel.refresh(false);
+			// Set controllerInfo.name if the user did not set it.
```

Comment:
This comment isn't really helpful - a better one would be `Default to the name of the controller instance if the user did not specify their own`

Reply:
Fixed. Thank you.